### PR TITLE
Adding content type to file repository and clarifying version of NIFTI for files

### PR DESCRIPTION
**Pull request by Peyman-N**

### CODE REVIEW

**Review by lzehl — APPROVED** [review]
@apdavison looks okay to me

**Comment by lzehl** [comment]
only comment: for not trying to load dummy files instead of setting an arbitrary file size you could also make use of the try/except functionality in python

File: bids2openminds/utility.py [hunk 1]
```diff
@@ -140,5 +141,49 @@ def file_hash(file_path: str, algorithm: str = "MD5"):
...
+                    return ContentType.by_name("application/vnd.nifti.1")
+                elif sizeof_hdr == nii2_sizeof_hdr:
+                    return ContentType.by_name("application/vnd.nifti.2")
+                else:  # big endian
+                    sizeof_hdr = int.from_bytes(byte_data, byteorder='big')
+                    if sizeof_hdr == nii1_sizeof_hdr:
+                        return ContentType.by_name("application/vnd.nifti.1")
+                    elif sizeof_hdr == nii2_sizeof_hdr:
+                        return ContentType.by_name("application/vnd.nifti.2")
+    except:
```
Comment (apdavison) [inline bids2openminds/utility.py:188]: you should never use a bare `except`. What kind of exception(s) can happen here?

File: bids2openminds/main.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 import openminds.latest.controlled_terms as controlled_terms
 from openminds import IRI
 
-from .utility import table_filter, pd_table_value, file_hash, file_storage_size
+from .utility import table_filter, pd_table_value, file_hash, file_storage_size, detect_nifti_verstion
```
Comment (apdavison) [inline bids2openminds/main.py:13]: typo: verstion --> version

**Review by apdavison — CHANGES_REQUESTED** [review]

**Pull request merged**